runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatConfig.java

```diff
@@ -122,6 +123,7 @@ default boolean alwaysShowBaseStats()
 		return false;
 	}
 
+	@Alpha
```

Comment:
I'm not sure we want to draw transparent text on the tooltip since that could look very odd.

Reply:
Agreed. I'm going through all places where I added alpha to check if the colors are also used for text, haven't gotten to this one just yet though :wink: